htdocs/contact/card.php

```diff
@@ -105,6 +106,8 @@
 if ($reshook < 0) {
 	setEventMessages($hookmanager->error, $hookmanager->errors, 'errors');
 }
+// Actions cancel, add, update, delete or clone
+include DOL_DOCUMENT_ROOT.'/core/actions_addupdatedelete.inc.php';
```

Comment:
Because we already have the code of action "inline" into the contact/card.php file, why adding the 
`include DOL_DOCUMENT_ROOT.'/core/actions_addupdatedelete.inc.php';`
We must choose one or the other.
Because we have specific code, we should keep working with the exisiing code
`if ($action == 'add' && $user->rights->societe->contact->creer) {`